src/test/java/com/klarna/hiverunner/builder/HiveShellBaseTest.java

```diff
@@ -75,109 +84,109 @@ public void setupScriptsShouldBeExecutedAtStart() throws IOException {
 
-      verify(container).executeScript(hql);
+      verify(container).executeStatement(hiveSql);
     }
 
     @Test
     public void executeScriptCharsetFile() throws IOException {
-      String hql = "use default;";
+      String hiveSql = "use default";
 
-      File file = new File(tempFolder.getRoot(), "script.hql");
-      Files.write(hql, file, UTF_8);
+      File file = new File(tempFolder.getRoot(), "script.sql");
+      Files.write(hiveSql, file, UTF_8);
 
-      HiveShell shell = createHiveShell();
+      HiveShell shell = createHiveCliShell();
       shell.start();
       shell.execute(UTF_8, file);
 
-      verify(container).executeScript(hql);
+      verify(container).executeStatement(hiveSql);
     }
     
     @Test
     public void executeScriptPath() throws IOException {
-      String hql = "use default;";
+      String hiveSql = "use default";
 
-      File file = new File(tempFolder.getRoot(), "script.hql");
-      Files.write(hql, file, UTF_8);
+      File file = new File(tempFolder.getRoot(), "script.sql");
+      Files.write(hiveSql, file, UTF_8);
 
-      HiveShell shell = createHiveShell();
+      HiveShell shell = createHiveCliShell();
       shell.start();
       shell.execute(Paths.get(file.toURI()));
 
-      verify(container).executeScript(hql);
+      verify(container).executeStatement(hiveSql);
     }
 
     @Test
     public void executeScriptCharsetPath() throws IOException {
-      String hql = "use default;";
+      String hiveSql = "use default";
 
-      File file = new File(tempFolder.getRoot(), "script.hql");
-      Files.write(hql, file, UTF_8);
+      File file = new File(tempFolder.getRoot(), "script.sql");
+      Files.write(hiveSql, file, UTF_8);
 
-      HiveShell shell = createHiveShell();
+      HiveShell shell = createHiveCliShell();
       shell.start();
       shell.execute(UTF_8, Paths.get(file.toURI()));
 
-      verify(container).executeScript(hql);
+      verify(container).executeStatement(hiveSql);
     }
 
     @Test(expected = IllegalArgumentException.class)
     public void executeScriptFileNotExists() throws IOException {
-      File file = new File(tempFolder.getRoot(), "script.hql");
+      File file = new File(tempFolder.getRoot(), "script.sql");
 
-      HiveShell shell = createHiveShell();
+      HiveShell shell = createHiveCliShell();
       shell.start();
       shell.execute(UTF_8, Paths.get(file.toURI()));
     }
     
     @Test(expected = IllegalStateException.class)
     public void executeScriptNotStarted() throws IOException {
-      File file = new File(tempFolder.getRoot(), "script.hql");
+      File file = new File(tempFolder.getRoot(), "script.sql");
       
-      HiveShell shell = createHiveShell();
+      HiveShell shell = createHiveCliShell();
       shell.execute(UTF_8, Paths.get(file.toURI()));
     }
     
 	@Test
 	public void executeQueryFromFile() throws IOException {
-		HiveShell shell = createHiveShell();
+		HiveShell shell = createHiveCliShell();
```

Comment:
I'm wishing now I'd never looked at the indentation, within this file there is 2 and 8. If 4 makes the most sense overall for the project please update accordingly....